yal/main.py

```diff
@@ -128,8 +128,19 @@ async def process_message(self, message):
             message.session_event = Message.SESSION_EVENT.RESUME
             self.state_name = feedback_state
 
+        # Replies to template push messages
+        payload = utils.get_by_path(
+            message.transport_metadata, "message", "button", "payload"
+        )
+        if payload and payload.startswith("state_") and payload in dir(self):
+            self.user.session_id = None
+            self.state_name = payload
```

Comment:
Shouldn't this direct the user to `state_testing_payload`?

Reply:
I was thinking that for the production test, we make the payload `state_testing_payload`, since all the other states that it would normally direct to aren't in this deploy. I also wanted to be able to test the actual routing functionality in production, not just hardcode it to always go to the same state.

Reply:
@rudigiesler but the intention of this is to allow us to test the template messages? I'd prefer to ensure that the states are being added to the payload correctly. And that different buttons send different payloads.
That also means that I can add the correct payloads to the templates now and I don't have to go through the flows to update the payloads when we need to deploy

Reply:
Ah I see what you're saying. My intention with this was to be able to test this bit of code, that we receive the payload correctly from whatsapp, and that the routing to a state code works as expected, I wasn't thinking of testing the actual template sending webhooks.

What about if we do both? If I add the same states that we have in the main branch, but here they're just an endstate, instead of being an assessment?